Packs/KeeperSecretsManager/Integrations/KeeperSecretsManagerEventCollector/KeeperSecretsManagerEventCollector.py

```diff
@@ -0,0 +1,540 @@
+import urllib3
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+from keepercommander import api
+from keepercommander.params import KeeperParams
+from keepercommander.auth.login_steps import LoginStepDeviceApproval, DeviceApprovalChannel, LoginStepPassword
+from keepercommander import utils, crypto
+from keepercommander.loginv3 import LoginV3Flow, LoginV3API, InvalidDeviceToken
+from keepercommander.proto import APIRequest_pb2
+
+""" CONSTANTS """
+
+VENDOR = "Keeper"
+PRODUCT = "Secrets Manager"
+LOG_LINE = f"{VENDOR}_{PRODUCT}:"
+DEFAULT_MAX_FETCH = 10000
+API_MAX_FETCH = 1000
+SESSION_TOKEN_TTL = 3600  # In seconds
+REGISTRATION_FLOW_MESSAGE = (
+    "In order to authorize the instance, first run the command `!ksm-event-collector-register-start`."
+    " A code will be sent to your email, copy it and paste that value in the command"
+    " `!ksm-event-collector-register-complete` as an argument to finish the process."
+)
+DEVICE_ALREADY_REGISTERED = (
+    "Device is already registered, try running the 'ksm-event-collector-register-complete'"
+    " command without supplying a code argument."
+)
+""" Fetch Events Classes"""
+LAST_RUN = "Last Run"
+
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+""" CONSTANTS """
+
+DATE_FORMAT = "%Y-%m-%dT%H:%M:%SZ"  # ISO8601 format with UTC, default in XSOAR
+
+""" HELPER FUNCTIONS """
+
+
+def get_max_events_to_fetch(params_max_fetch: str | int) -> int:
+    """Gets the maximum number of events to fetch.
+
+    Args:
+        params_max_fetch (str): A limit for the log type.
+
+    Returns:
+        int: The maximum number of events to fetch.
+
+    Raises:
+        ValueError: If the limit is not a valid integer.
+    """
+    try:
+        return DEFAULT_MAX_FETCH if params_max_fetch in ["", None] else int(params_max_fetch)
+
+    except ValueError:
+        raise ValueError("Please provide a valid integer value for a fetch limit.")
+
+
+def get_current_time_in_seconds() -> int:
+    """A function to return time as an int number of seconds since the epoch
+
+    Returns:
+        int: Number of seconds since the epoch
+    """
+    return int(time.time())
```

Comment:
I'll change it to be **int(datetime.now().timestamp())**, but I put it in a seperate function since we call it in two places and didn't want to have duplicate code.